website/pages/blog/cloudquery_tailscale.mdx

```diff
@@ -38,8 +45,9 @@ kind: source
 # Common source-plugin configuration
 spec:
   name: tailscale
-  path: cloudquery/tailscale
-  version: "VERSION_SOURCE_TAILSCALE"
+  path: /path/to/downloaded/plugin # Buy from here: https://cloudquery.io/integrations/tailscale
+  registry: local
+  version: "VERSION_PREMIUM"
```

Comment:
Not sure about these... just resolves to `version: "PREMIUM"` currently

Reply:
Yeah, premium plugins don't really have SemVer versions. This is similar to the `Render` and `Jira` plugins.
I don't mind what we put there (I initially put `v1.0.0`) as long as we're consistent

Reply:
But this should also be `"PREMIUM"` and not `"VERSION_PREMIUM"`